lib/appmap/rails/request_handler.rb

```diff
@@ -59,16 +39,18 @@ def to_h
               headers: headers,
             }.compact
 
-            h[:message] = params.keys.map do |key|
-              val = params[key]
-              {
-                name: key,
-                class: val.class.name,
-                value: self.class.display_string(val),
-                object_id: val.__id__,
-              }.tap do |message|
-                properties = object_properties(val)
-                message[:properties] = properties if properties
+            unless params.blank?
+              h[:message] = params.keys.map do |key|
```

Comment:
I've always wondered why it's `message`. Why not just pass HTTP parameters in the `parameters` attribute?

Reply:
`message` is understood by the UI code.